README.md

```diff
@@ -144,7 +147,7 @@ Echo.leaveChannel("private:channel2")
 Echo.join('channel3'); 
 Echo.leaveChannel("presence:channel3")
 ```
-instead of [Pusher Channel Conventions](https://pusher.com/docs/channels/using_channels/channels/#channel-types)
+instead of [Pusher Channel Conventions](https://pusher.com/docs/channels/using_channels/channels/#channel-types).
```

Comment:
This is confusing, in a sense how channels are created in the first place. 
They don't have a prefix, yet this example can put you into a dilemma whether the channel was created with the given prefix.
So, we also need to provide how we create a channel

```
 // public channel
Echo.channel('channel1');
Echo.leaveChannel("public:channel1");
// private channel
Echo.private('channel2'); 
Echo.leaveChannel("private:channel2")
// presence channel
Echo.join('channel3'); 
Echo.leaveChannel("presence:channel3")
```

I am not in favor of mixing `pusher` conventions with `ably`, it's like mixing wrong and right code together.

Reply:
Also, I am not in favor of mixing `pusher` conventions with `ably`, it's like mixing wrong and right code together.

Reply:
You can check official doc => 
https://laravel.com/docs/9.x/broadcasting#leaving-a-channel

This information is not enough at all. Leaving the channel section is confusing, we are trying to simplify it for both `pusher` and `ably`.

Reply:
ok that makes sense thanks. I still think having two large snippets to explain this is quite awkward to read. how about this?

```js
 // For public channels
Echo.channel("channel1");
// Use this:
Echo.leaveChannel("public:channel1");
// Instead of:
Echo.leaveChannel("channel1");

// etc...
```

Reply:
done